core/src/reporter/utils.ts

```diff
@@ -177,15 +177,15 @@ export async function sendTransactionDelegatedFee({
   const endpoint = buildUrl(ORAKL_NETWORK_DELEGATOR_URL, `sign`)
 
   let response
+
   try {
     response = (
-      await axios.post(endpoint, {
-        ...transactionData
-      })
+      await axios.post(endpoint, { ...transactionData }, { timeout: Number(DELEGATOR_TIMEOUT) })
```

Comment:
@nick-bisonai I think we should convert the `DELEGATOR_TIMEOUT` inside of settings.

```suggestion
      await axios.post(endpoint, { ...transactionData }, { timeout: DELEGATOR_TIMEOUT })
```


settings.ts

```
export const DELEGATOR_TIMEOUT = Number(process.env.DELEGATOR_TIMEOUT) || 3000
```